djstripe/models/base.py

```diff
@@ -380,6 +385,32 @@ def _create_from_stripe_object(
 
         return instance
 
+    @classmethod
+    def get_or_create(cls, **kwargs):
+        """ Get or create an object of this class."""
+
+        try:
+            return cls.objects.get(id=kwargs["id"]), False
+        except cls.DoesNotExist:
+            return cls.create(**kwargs), True
+
+    @classmethod
+    def create(cls, **kwargs):
+        api_kwargs = dict(kwargs)
+
+        for k, v in api_kwargs.items():
+            if k != "object":  # We want to ignore the Stripe API's object attribute
```

Comment:
@therefromhere Thoughts on this methodology? It's only the "object" attribute from the Stripe API we want to ignore as you're inside the correct class already.

Reply:
Does the object param cause a problem?

Reply:
Yes it does, 'object' is not an attribute of any of the StripeModels so cls._meta.get_field('object') fails https://travis-ci.org/dj-stripe/dj-stripe/builds/578801734